LuaUI/Widgets/api_gadget_icons.lua

```diff
@@ -214,6 +221,24 @@ function SetIcons()
 			end
 		end
 
+		local padExcluded = excludedPadUnitDef[unitDefID] and Spring.GetUnitRulesParam(unitID, "padExcluded")
```

Comment:
This should probably be done via widget icons and not gadget icons because a unit rules param does not discriminate between teams

Reply:
Whether a pad is excluded is determined by gadgets, not widgets, and remembering the state that was sent wouldn't survive luaui reload. Why not just append a teamID to padExcluded?

Reply:
Sounds good

Reply:
Do you mean setting the unitrulesparam value to the teamID that wants to see it?

Reply:
No, incorporating the teamID into the param key like this:
```suggestion
		local padExcluded = excludedPadUnitDef[unitDefID] and Spring.GetUnitRulesParam(unitID, "padExcluded" .. Spring.GetMyTeamID())
```